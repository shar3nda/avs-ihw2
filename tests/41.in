&&Q9/&UN<Z@G.f(= %nf#Z&wl-82N?&(r*gb6X
Dy,c6'*F_ i7-{$h^,.5Z39A	UXcf[}-?Fh3xA|