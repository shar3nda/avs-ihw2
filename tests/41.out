Lowercase: 95, uppercase: 108